src/components/DatePickerModal/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Header, HeaderTitle, StyledModal, StyledCalendar } from './styled';
+
+export default function DatePickerModal(props) {
+    const {
+        id,
+        className,
+        style,
+        locale,
+        minDate,
+        maxDate,
+        selectionType,
+        variant,
+        value,
+        onChange,
+        onRequestClose,
+        isOpen,
+        title,
+    } = props;
+
+    return (
+        <StyledModal
+            id={id}
+            className={className}
+            style={style}
+            isOpen={isOpen}
+            onRequestClose={onRequestClose}
+        >
+            <Header>
+                <HeaderTitle>{title}</HeaderTitle>
+            </Header>
```

Comment:
let's put this inside a RenderIf in order to avoid render markup when the title is not passed